### DISCUSSION TITLE
Markor v2.9.0 Update Release

### DISCUSSION THREAD

**Discussion by gsantner** [gsantner/markor#1701] [General] [answered]
# [Markor v2.9 - Snippets, Templates, Graphs, Charts, Diagrams, YAML front-matter, Chemistry](https://gsantner.net/blog/2022/05/14/markor-v2.9-snippets-zimwiki-math-daynight-privacy-markdown-todotxt.html)

See project post with highlighted changes:   
<https://gsantner.net/blog/2022/05/14/markor-v2.9-snippets-zimwiki-math-daynight-privacy-markdown-todotxt.html>

<hr /><em>This discussion was created from the release <a href='https://github.com/gsantner/markor/releases/tag/v2.9.0'>Markor v2.9.0</a>.</em>

**Comment by pbienst** [comment]
Thanks for the new release! I updated from the Google play version.

Could I ask to bring the 'exit' button (arrow to the left) back? It was really convenient to quickly go back to where I was. Perhaps the reason it was removed was to make space for the 'save' icon, but given the autosave feature, that's not really something I need...

Thanks for considering this!

**Reply by gsantner** [reply]
I regularly got emails and discussions here on GitHub what this button does, it was always confusing to people to understand ... whats the difference between it and the system back button.

And exactly, it was removed to get more space at the top bar - but mainly to see more of the file/document name.

It's gone for the good. Just use the system back button :smile: .

**Comment by pbienst** [comment]
I don't have a physical back button, and in order to perform a back action, I need to change my grip on the phone and perform an awkward swipe.

My workflow consists of making small changes to files many times throughout the day, and the loss of the icon really slows me down...

Can't you at least make the appearance of the back icon configurable? 

Or add an option to have the save button perform a back action after save? 

Otherwise I'll have to stick to 2.8...

Thanks!

**Comment by aubreyz** [comment]
Even with a physical back button it now takes 2 keypresses the get back instead of  1 with the top button. First press just closes the keyboard.

**Comment by aubreyz** [comment]
Surely what is needed is a button to save and go to file manager - not really a "back" button

**Reply by pbienst** [reply]
Personally, I don't want to go to file manager, but to the screen I was before. This could be the file manager, but also e. g. the home screen where I opened a markdown file from the widget. The back arrow in 2.8 did just that.

**Comment by achrodo** [comment, marked as answer]
Enjoying the new release. Thanks for your work! I was getting a little concerned because F-droid took longer than usual to update it, but all is well.

Q: The automatic printing of the current date on new lines in todo.txt — is there a way to disable that? It wasn't present for me in 2.8.

**Reply by gsantner** [reply]
Top menu File settings -> Auto-Format

This feature is available since the first Markor release with todo.txt support.

**Comment by acdel123** [comment]
Thanks for the new version. Looking forward to trying out the new snippets / templates.

One quick question though, Anyone else notice a problem with display of dollar signs in .md files in the new version?

When there is a just one $ on a line, it displays as it should. But if there is more than one on a line, something like...

Apples are normally $1.99 a pound, on sale today for the bargain price of $0.99

neither one displays in view mode.

Is this possibly related to https://github.com/gsantner/markor/pull/1576 ?

**Comment by gsantner** [comment]
Do you have Math/KaTex enabled?

**Comment by acdel123** [comment]
The Math/KaTex box is unchecked

**Comment by kal60** [comment]
Congrats for the new Release ! :-)
It's a great program.
Something regarding ZIM has been fixed obviously, so may be you could update the template for ZIM reference as well (see screenshot - combined styles)
cheers.
Knut
![SnapMarkup_20224007_024054](https://user-images.githubusercontent.com/8725410/177776572-e1b4ed6c-9054-4b01-a952-aff01ceb4b71.png)

**Reply by gsantner** [reply]
Do you mind to update the file (Markor repository samples folder) and make a merge request? Thank you!

**Reply by kal60** [reply]
I'll do that (when I'm back from vacation ;-) )
